microservices/CFDP/lib/cfdp_pdu/cfdp_pdu_finished.rb

```diff
@@ -21,6 +21,9 @@ def self.decom_finished_pdu_contents(pdu, pdu_hash, variable_data)
     pdu_hash["CONDITION_CODE"] = s.read("CONDITION_CODE")
     pdu_hash["DELIVERY_CODE"] = s.read("DELIVERY_CODE")
     pdu_hash["FILE_STATUS"] = s.read("FILE_STATUS")
+    if pdu_hash['VERSION'] == 0
+      pdu_hash["END_SYSTEM_STATUS"] = s.read("END_SYSTEM_STATUS")
+    end
```

Comment:
At line 65 you're appending END_SYSTEM_STATUS in both version 0 and 1. Is this not SPARE here in version 1?

Reply:
It is spare in version 1, which is why it isn't read out in the decom method.